<ui version="4.0" >
 <class>MainDialogClass</class>
 <widget class="QMainWindow" name="MainDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>410</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>410</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MEG Video Recording Station</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="startButton" >
    <property name="geometry" >
     <rect>
      <x>310</x>
      <y>20</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>310</x>
      <y>60</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="localGroupBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title" >
     <string>Local audio</string>
    </property>
    <widget class="QProgressBar" name="senderLevelLeft" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>30</y>
       <width>10</width>
       <height>95</height>
      </rect>
     </property>
     <property name="value" >
      <number>24</number>
     </property>
     <property name="textVisible" >
      <bool>false</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="senderLevelRight" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>30</y>
       <width>10</width>
       <height>95</height>
      </rect>
     </property>
     <property name="value" >
      <number>24</number>
     </property>
     <property name="textVisible" >
      <bool>false</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Sent to the remote site</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>130</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>L</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>130</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>R</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="remoteGroupBox" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>10</y>
      <width>111</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title" >
     <string>Remote audio</string>
    </property>
    <widget class="QProgressBar" name="receiverLevelLeft" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>30</y>
       <width>10</width>
       <height>95</height>
      </rect>
     </property>
     <property name="value" >
      <number>24</number>
     </property>
     <property name="textVisible" >
      <bool>false</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="suffixEdit" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>220</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Savefile suffix</string>
    </property>
    <property name="wordWrap" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialogClass</receiver>
   <slot>onStartRec()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>56</x>
     <y>993</y>
    </hint>
    <hint type="destinationlabel" >
     <x>649</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialogClass</receiver>
   <slot>onStopRec()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>176</x>
     <y>993</y>
    </hint>
    <hint type="destinationlabel" >
     <x>649</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStartRec()</slot>
  <slot>onStopRec()</slot>
  <slot>onShutterChanged(int)</slot>
  <slot>onGainChanged(int)</slot>
  <slot>onCam1Toggled(bool)</slot>
  <slot>onCam2Toggled(bool)</slot>
  <slot>onExit()</slot>
 </slots>
</ui>
